includes/admin/views/html-admin-page-status-logs-db.php

```diff
@@ -6,8 +6,17 @@
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
+$s = '';
+if ( isset( $_REQUEST['s'] ) ) {
+	$s = wc_clean( $_REQUEST['s'] );
```

Comment:
Missing `wp_unslash()` before sanitization.

Reply:
Done.

Reply:
It's not done, still missing `wp_unslash()` here.

Reply:
Now should be done :)

Reply:
@seostudio it's still not done :p Look at the diff above.

Reply:
@mikejolley I apologize for the misunderstanding. Now is OK?